src/theme.ts

```diff
@@ -80,9 +80,11 @@ interface Variations {
 interface Background {
   dark: string
   selected: string
+  newDesign?: States
```

Comment:
Please change the name to more meaningful

Reply:
changed to 'tracked'